src/components/content/filter-line/FilterLine.tsx

```diff
@@ -1,19 +1,33 @@
 import "./FilterLine.css";
 import GenreSelect from "./genre-select/GenreSelect";
 import SortControl from "./sort-control/SortControl";
-import {FilterData} from "../../../models/filter";
 import {GenreTitle} from "../../../constants/genres-const";
 import React from "react";
+import useMultipleSearchParams from "../../../hooks/UseMultipleSearchParams";
+
+function FilterLine(): React.JSX.Element {
+    const { updateQueryParams } = useMultipleSearchParams();
+    const urlSearchParams = new URLSearchParams(location.search);
+    const {genre, sort} = Object.fromEntries(urlSearchParams.entries());
```

Comment:
Can you move this and 'new URLSearchParams' functionalities into 'useMulitpleSearchParams' hook?

Reply:
Sorry, I don't understand why to move this part responsible for reading url params into hook which is responsible for saving url params